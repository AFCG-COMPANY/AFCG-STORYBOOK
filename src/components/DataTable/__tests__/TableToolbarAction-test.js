import React from 'react';
import { mount } from 'enzyme';
import { TableToolbarAction } from '../';

describe('DataTable.TableToolbarAction', () => {
  it('should render', () => {
    const wrapper = mount(
      <TableToolbarAction
        className="custom-class"
        iconName="add--solid"
        iconDescription="Add"
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
