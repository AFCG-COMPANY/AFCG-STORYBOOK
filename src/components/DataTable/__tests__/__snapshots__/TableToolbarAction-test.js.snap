// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable.TableToolbarAction should render 1`] = `
<TableToolbarAction
  className="custom-class"
  iconDescription="Add"
  iconName="add--solid"
>
  <button
    className="custom-class bx--toolbar-action"
  >
    <Icon
      className="bx--toolbar-action__icon"
      description="Add"
      fillRule="evenodd"
      name="add--solid"
      role="img"
    >
      <svg
        alt="Add"
        aria-label="Add"
        className="bx--toolbar-action__icon"
        fillRule="evenodd"
        height="16"
        name="add--solid"
        role="img"
        viewBox="0 0 16 16"
        width="16"
      >
        <title>
          Add
        </title>
        <path
          d="M7 7H4v2h3v3h2V9h3V7H9V4H7v3zm1 9A8 8 0 1 1 8 0a8 8 0 0 1 0 16z"
          key="key0"
        />
      </svg>
    </Icon>
  </button>
</TableToolbarAction>
`;
